### PR TITLE
It allows max buffer when building for production

### PR DESCRIPTION
**Pull request by paolocarrasco**
- It can be configured either in the command line or in the configuration file.
- It fixes the problem with https://github.com/SoCreate/angular-playground/issues/155

### CODE REVIEW

**Comment by paolocarrasco** [comment]
Changes were performed @timstoddard, could you please take a look?

File: packages/angular-playground/cli/src/build-angular-cli.ts [hunk 1]
```diff
@@ -9,11 +9,16 @@ export async function buildAngularCli(appName: string, baseHref: string) {
...
 
     console.log('Building for production with sandboxes...');
+    console.log(`Max buffer set? ${maxBuffer ? +maxBuffer : 'No'}`);
+
     // Cannot build w/ AOT due to runtime compiler dependency
-    exec(`ng build ${appName} --prod --aot=false --base-href=${baseHref}`, (err, stdout, stderr) => {
-        if (err) throw err;
-        console.log(stdout);
-    });
+    const options = maxBuffer && Number.isInteger(+maxBuffer) ? {maxBuffer: +maxBuffer} : {};
```
[the remark this answers is not in the document]
Comment (timstoddard) [reply packages/angular-playground/cli/src/build-angular-cli.ts]: Please also update the type of the `maxBuffer` param to be `number | string` to reflect what you stated above. Then I will approve the PR.

**Comment by timstoddard** [comment]
@paolocarrasco just one more thing, then we should be good to merge in the PR.

**Comment by paolocarrasco** [comment]
Thanks @timstoddard ! All is good now. Can we merge this or assign it to someone who can do it?

**Pull request merged**